modules/lvms-creating-lvms-cluster-using-cli.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-lvms-cluster-using-cli_{context}"]
+= Creating a Logical Volume Manager cluster using the CLI
+
+You can create a Logical Volume Manager cluster on a worker node using the {product-title} CLI.
+
+.Prerequisites
+
+* You have installed the {product-title} CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have installed {lvms}.
+
+* You have installed a worker node in the cluster.
+
+.Procedure
+
+. Save the `LVMCluster` custom resource (CR) YAML in a file.
++
+Specify a name for the file. For example, `lvmcluster.yaml`.
+
+. Create the `LVMCluster` CR:
++
+[source,terminal]
+----
+$ oc create -f <file-name> <1>
+----
+<1> Replace `<file-name>` with the name of the `LVMCluster` CR YAML file.
++
+.Example output
+[source,terminal]
+----
+lvmcluster/lvmcluster created
+----
++
+[IMPORTANT]
+====
+You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
+====
++
+The `LVMCluster` resource creates the following system-managed CRs:
++
+`LVMVolumeGroup`:: Tracks individual volume groups across multiple nodes. It provides the back-end storage for the persistent volumes that you create.
++
+It is a specific type of persistent volume (PV) that is backed by an LVM Volume Group.
+`LVMVolumeGroupNodeStatus`:: Tracks the status of the volume groups on a node.
+
+[NOTE]
+====
+When you create a Logical Volume Manager cluster, `StorageClass` and `LVMVolumeGroup` resources work together to provide dynamic provisioning of storage.
+`StorageClass` CRs define the properties of the storage that you can dynamically provision.
+====
+
+.Verification
+
+Verify that the `LVMCluster` resource has created the `StorageClass`, `LVMVolumeGroup`, and `LVMVolumeGroupNodeStatus` CRs.
+
+[IMPORTANT]
+====
+`LVMVolumeGroup` and `LVMVolumeGroupNodeStatus` are managed by {lvms}. Do not edit these CRs directly.
+====
+
+. Check that the `LVMCluster` CR is in `ready` state:
++
+[source,terminal]
+----
+$ oc get lvmclusters.lvm.topolvm.io -o jsonpath='{.items[*].status.deviceClassStatuses[*]}'
+----
++
+.Example output
+[source,json]
+----
+{
+    "name": "vg1",
+    "nodeStatus": [
+        {
+            "devices": [
+                "/dev/nvme0n1",
+                "/dev/nvme1n1",
+                "/dev/nvme2n1"
+            ],
+            "node": "kube-node",
+            "status": "Ready"
+        }
+    ]
+}
+----
+
+. Check that the storage class is created:
```

Comment:
All the verification steps under this one are unnecessary. LVMCluster CR will already signal an error if there was a failure while creating these resources.

Reply:
Verification info was added as a part of - https://github.com/openshift/openshift-docs/pull/56366
Is this info useful to the users at any point? Or should we get rid of this completely?

Reply:
@suleymanakbas91 Thank you the update.
Updated the content as per the suggestion. PTAL.
Thanks.